src/main/c/Makefile11

```diff
@@ -43,9 +43,5 @@ all: $(TARGET)
 $(TARGET): $(JNI_HEADERS) $(JNI_SOURCES)
```

Comment:
Do we need to remove `JNI_HEADERS` from here as well?

Reply:
done - thx!